geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DataCommands.java

```diff
@@ -736,7 +736,7 @@ public Result exportData(
       @CliOption(key = CliStrings.EXPORT_DATA__FILE,
           unspecifiedDefaultValue = CliMetaData.ANNOTATION_NULL_VALUE, mandatory = true,
           help = CliStrings.EXPORT_DATA__FILE__HELP) String filePath,
-      @CliOption(key = CliStrings.EXPORT_DATA__MEMBER,
+      @CliOption(key = CliStrings.MEMBER,
```

Comment:
This makes me wonder if we should file a Jira ticket to make sure all Gfsh commands with MEMBER option are consistently allowing comma-delimited lists of members.